lib/systems/src/PedalsSystem.cpp

```diff
@@ -67,7 +67,7 @@ bool PedalsSystem::evaluate_pedal_implausibilities_(const AnalogConversion_s &pe
     // FSAE T.4.2.10
     bool pedal1_swapped = false;
     bool pedal2_swapped = false;
-    float margin_of_error = 0.10;
```

Comment:
This should be overwritten with PedalsSystem changes, but make sure this is updated to the value that you expect when you merge in. We were unable to trigger implausibility manually on some pedals, so make sure that this is not too much margin of error. This is worth deliberately testing again ofc